src/routes/home/components/announcementDialog/data/announcementData.js

```diff
@@ -1,13 +1,20 @@
+// import { Card } from '../../../../../../tests/e2e/fixtures/cards';
+
 export const announcementData = {
-  id: 'opponentHandRevealed',
+  id: 'spades2024Announcement',
   activatorText: 'announcement.activatorText',
   title: 'announcement.title',
-  imgSrc: './img/announcement/deck-exhausted.gif',
   displayCards: [],
+  imgSrc: '/img/announcement/spades-2024-bracket.svg',
+  startTime: '2025-01-24',
+  endTime: '2025-01-26',
```

Comment:
The intent is to allow defining announcements in advance of when they go live, and to give them an expiration date so they don't require code changes to take down